contents/blog/feature-flags-vs-configuration.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "Feature flags vs configuration: what should you choose?"
+date: 2023-06-27
+author: ["ian-vanagas"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+category: Engineering
+tags:
+ - Guides
+ - Feature flags
+ - Explainers
+---
+
+Sometimes you want to change the behavior of your app without changing the code. Both feature flags and configurations are options for doing this. 
+
+In their most basic form, they are a key you add to your code that returns a value you can control. You can use this value to change the behavior or experience of your app or product.
+
+This post goes over what configuration and feature flags do, and which one you should choose.
```

Comment:
```suggestion
Sometimes you want to change the behavior of your app without changing the code. Both feature flags and configurations are options for doing this, but it may not be clear which is the right option to use.

This post compares configurations and feature flags, and how to choose the right one for your use case.
```